tox.ini

```diff
@@ -24,9 +24,6 @@ commands =
     generate_validation: python setup.py generate_validation
     # TODO(rhcarvalho): check syntax of other important entrypoint playbooks
     ansible_syntax: python setup.py ansible_syntax
-    # ansible 2.2.2+ unfortunately breaks the integration test runner
-    # because it can no longer set facts on the test docker hosts.
-    # So for now, install separate ansible version for integration.
     # PR that fixes it: https://github.com/ansible/ansible/pull/23599
     # Once that PR is available, drop this and use same ansible.
-    integration: pip install ansible==2.2.1.0
+    integration: pip install ansible==2.3.1.0
```

Comment:
no real reason to keep this or the comment above. could replace with:

`    integration: python -c 'print "run test/integration/run-tests.sh"'`